src/operator-docs/transformation/expand.ts

```diff
@@ -1,6 +1,79 @@
 import { OperatorDoc } from '../operator.model';
 
 export const expand: OperatorDoc = {
-  'name': 'expand',
-  'operatorType': 'transformation'
+  name: 'expand',
+  operatorType: 'transformation',
+  signature:
+    'public expand(project: function(value: T, index: number), concurrent: number, scheduler: Scheduler): Observable',
+  parameters: [
+    {
+      name: 'project',
+      type: 'function(value: T, index: number)',
+      attribute: '',
+      description: `A function that, when applied to an item emitted by the source or the output Observable, returns an Observable.`
+    },
+    {
+      name: 'concurrent',
+      type: 'number',
+      attribute: 'optional default: Number.POSITIVE_INFINITY',
+      description: `Maximum number of input Observables being subscribed to concurrently.`
+    },
+    {
+      name: 'scheduler',
+      type: 'Scheduler',
+      attribute: 'optional default: null',
+      description: `The IScheduler to use for subscribing to each projected inner Observable.`
+    }
+  ],
+  useInteractiveMarbles: true,
+  marbleUrl: 'http://reactivex.io/rxjs/img/expand.png',
+  shortDescription: {
+    description: `Recursively projects each source value to an Observable which is merged in the output Observable.`
+  },
+  walkthrough: {
+    description: `<p>Returns an Observable that emits items based on applying a function
+     that you supply to each item emitted by the source Observable,
+     where that function returns an Observable,
+     and then merging those resulting Observables and
+     emitting the results of this merger. <i>Expand</i> will re-emit on the output
+     Observable every source value.
+     Then, each output value is given to the <code>project</code> function
+     which returns an inner Observable to be merged on the output Observable.
+     Those output values resulting from the projection are also given to the
+     <code>project</code> function to produce new output values.
+     This is how <i>expand</i> behaves recursively.</p>`
+  },
+  examples: [
+    {
+      name:
+        'Start emitting the powers of two on every click, at most 10 of them',
+      code: `
+      import { fromEvent, of } from 'rxjs/observable/fromEvent';
```

Comment:
I think you were looking for this import (also), 
`import { of } from 'rxjs/observable/Of';`

Awesome job, thanks for your pr.

Reply:
as you did here,  https://stackblitz.com/edit/reactive-expand

Reply:
ah, forgot to add it in. will update accordingly. thanks!